src/Message/Message.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import Icon from '../Icon'
+import styles from './Message.module.scss'
+
+const Message = ({
+  wrapperClassName,
+  className,
+  variant,
+  children,
+  icon,
+  iconClassName,
+  ...props,
+}) => (
+  <div className={cx(wrapperClassName, styles.wrapper, styles[variant])} {...props}>
+    {icon && <Icon type={icon} className={cx(iconClassName)} />}
+    <p className={cx(className, styles.message, styles[variant])}>
```

Comment:
1. `className` should be applied at the top level element. In our case, it's the `div` at line `16`;
2. `message` class is absent in the stylesheet;
3.  This element logically differs from the `div` at line `16`, but it uses the same `styles[variant]` which breaks BEM;
4. There is no need for this element to contain a user-defined class.

**Bottom line:**
This element should not have `className` prop at all.
**Also it's better to make it as `div`, because `p` has default margin**